solutions/USACO feb18-teleportation.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: usaco-807
+source: USACO Bronze 2018 February
+title: Teleportation
+author: Ananth Kashyap
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_teleport_bronze_feb18.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(1)$
+
+<LanguageSection>
+
+<PySection>
+
+```python
+# Take in data using Python file i/o system
+file_in = open('teleport.in')
+data = file_in.read().strip().split('\n')
+a, b, x, y = map(int, data[0].split(' '))
+
+# Manually sort a, b to be increasing order
```

Comment:
```suggestion
# Swap a and b if a > b
```

Reply:
this is essentially literally what the code is doing; if you're going to change it to this, you might as well not have the comment at all (since people can tell from the code)

Reply:
I agree w maggie. It's fine as is.